install_config/router/default_haproxy_router.adoc

```diff
@@ -1498,18 +1498,45 @@ To verify the actual amount of ARP entries for IPv4, run the following:
 ----
 
 If the number begins to approach the `net.ipv4.neigh.default.gc_thresh3`
-threshold, increase the values. Run the following on the nodes running the
-router pods. The following sysctl values are suggested for clusters with large
-numbers of routes:
+threshold, increase the values. Get the current value by running:
 
 ----
-net.ipv4.neigh.default.gc_thresh1 = 8192
-net.ipv4.neigh.default.gc_thresh2 = 32768
-net.ipv4.neigh.default.gc_thresh3 = 65536
+# sysctl net.ipv4.neigh.default.gc_thresh1
+net.ipv4.neigh.default.gc_thresh1 = 128
+# sysctl net.ipv4.neigh.default.gc_thresh2
+net.ipv4.neigh.default.gc_thresh2 = 512
+# sysctl net.ipv4.neigh.default.gc_thresh3
+net.ipv4.neigh.default.gc_thresh3 = 1024
 ----
 
-To make these settings permanent across reboots, create a
-link:https://access.redhat.com/solutions/1305833[custom tuned profile].
+The following sysctl sets the variables to the {product-title} current default values.
+
+----
+# sysctl net.ipv4.neigh.default.gc_thresh1=8192
+# sysctl net.ipv4.neigh.default.gc_thresh2=32768
+# sysctl net.ipv4.neigh.default.gc_thresh3=65536
+----
+
+To make these settings permanent across reboots, until an {product-title} upgrade 
+occurs, edit the following file on each node:
+
+----
+/usr/lib/tuned/atomic-openshift-node-host/tuned.conf
```

Comment:
@eparis So we should tell the admin to copy /usr/lib/tuned/atomic-openshift-node-host/tuned.conf to /etc/tuned/atomic-openshift-node-host/tuned.conf and then add the lines. Note RHEL relies on files in /usr/lib/tuned/. Also, since the rpm puts the file in /usr/lib/tuned/atomic-openshift-node-host/tuned.conf we need to document the upgrade process to include verifying and maybe updating /etc/tuned/atomic-openshift-node-host/tuned.conf and when deleting the rpm we need to tell people to also delete /etc/tuned/atomic-openshift-node-host/tuned.conf. Failure to delete the file will mean that the RHEL defaults are not restored.

This all sounds very wrong to me.